nifi-asymmetrik-standard-bundle/nifi-asymmetrik-standard-reporting-tasks/src/main/java/com/asymmetrik/nifi/reporting/AbstractNiFiClusterMetricsReporter.java

```diff
@@ -12,6 +10,12 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
+import com.amazonaws.util.EC2MetadataUtils;
```

Comment:
Since these are AWS specific, none of these changes should be in the abstract class.  It may be better to subclass this abstract class with another AWS-specific abstract class.